charts/karmada-operator/templates/karmada-operator-deployment.yaml

```diff
@@ -37,12 +37,13 @@ spec:
         image: {{ template "karmada.operator.image" . }}
         imagePullPolicy: {{ .Values.operator.image.pullPolicy }}
         ports:
-        - containerPort: 8080
+        - containerPort: {{ .Values.service.port }}
           name: metrics
           protocol: TCP
         command:
         - /bin/karmada-operator
         - --leader-elect-resource-namespace={{ .Release.Namespace }}
+        - --metrics-bind-address={{ .Values.service.port }}
```

Comment:
This is just the port. We need to set the entire address, not just the port.

Reply:
Thanks @jabellard 
Got it
Fixed